assets/training/finetune_acft_image/environments/acft_image_huggingface/context/Dockerfile

```diff
@@ -12,4 +12,4 @@ RUN pip install -r requirements.txt --no-cache-dir
 RUN pip install gunicorn==22.0.0
 RUN pip install certifi==2024.07.04
 
-# dummy number to change when needing to force rebuild without changing the definition: 3
\ No newline at end of file
+# dummy number to change when needing to force rebuild without changing the definition: 4
```

Comment:
remove these lines from the docker context of **all** your environments

Reply:
This is needed because tag for base image is calculated like : `FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2204-cu118-py310-torch222:{{latest-image-tag:biweekly\.\d{6}\.\d{1}.*}}`  ; We need to move to latest tag of base image to resolve vulnerabilities ;  Can we continue with that dummy change, given we need latest tag ?

Reply:
no